languages/go/tree-sitter-go.cabal

```diff
@@ -25,7 +25,8 @@ library tree-sitter-go-internal
   build-depends:       base                 >= 4.12 && < 5
                      , tree-sitter ^>= 0.1.0.0
   default-language:    Haskell2010
-  Include-dirs:        vendor/tree-sitter-go/src
+  include-dirs:        vendor/tree-sitter-go/src
+  install-includes:    tree_sitter/parser.h
```

Comment:
Just curious - what does `install-includes` do?

Reply:
@maxbrunsfeld Copies that header to a shared place where FFI calls can access it.